I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 403, in script
    r "Oh iya kak mau ke toko buk—"
Exception: Sayer 'r' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 403, in script
    r "Oh iya kak mau ke toko buk—"
  File "C:\renpy\renpy-8.3.7-sdk\renpy\ast.py", line 2586, in execute
    Say.execute(self)
  File "C:\renpy\renpy-8.3.7-sdk\renpy\ast.py", line 583, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\renpy\renpy-8.3.7-sdk\renpy\ast.py", line 472, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'r' is not defined.

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.7.25031702
thelast 1.0
Thu Jun  5 20:29:13 2025
